ppmclibs/tinycv.pm

```diff
@@ -22,6 +22,7 @@ use warnings;
 use bmwqemu 'diag';
 
 use File::Basename;
+use Math::Complex 'sqrt';
```

Comment:
I think you could have avoided this import if you had saved the result of the square root calculated in map_error_to_avg. It's the same, isn't it? (just represented as percentage in the webui)